app/views/users/show.erb

```diff
@@ -0,0 +1,16 @@
+<div "user-show">
+  <h3>Hi! <%=@user.username%> Here are the surveys that you have completed:</h3>
+  <p> Click for stats </p>
+    <% @user_taken_surveys.each do |survey| %>
+    <ol>
+      <li><a href="/taken_survey/:<%=taken_survey.id%>/answers"><%= survey.name %>  </a></li>
```

Comment:
this could possibly be the logic for find the specific survey and its name Survey.find(taken_survey.survey_id).name
could also be an instance method in the model